src/SmartCommander/ViewModels/MainWindowViewModel.cs

```diff
@@ -39,8 +39,11 @@ public MainWindowViewModel()
             TabCommand = ReactiveCommand.Create(ChangeSelectedPane);
             OptionsCommand = ReactiveCommand.CreateFromTask(ShowOptions);
 
-            LeftFileViewModel = new FilesPaneViewModel(this) { IsSelected = true };
+            LeftFileViewModel = new FilesPaneViewModel(this);
+            LeftFileViewModel.FocusChanged += OnFocusChanged;
+
             RightFileViewModel = new FilesPaneViewModel(this);
+            LeftFileViewModel.FocusChanged += OnFocusChanged;
```

Comment:
Not sure, why you add FocusChanged event handler for LeftFileViewModel twice? Should it be RightFileViewModel in the second case?

Reply:
fixed